cmd/multiFaToVcf/multiFaToVcf.go

```diff
@@ -0,0 +1,43 @@
+package main 
+
+import(
+	"flag"
+	"fmt"
+	"log"
+	"github.com/vertgenlab/gonomics/convert"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/vcf"
+)
+
+func multiFaToVcf(inFile string, chr string, outFile string) {
+	f := fasta.Read(inFile)
+	v := convert.PairwiseFaToVcf(f, chr)
+	vcf.Write(outFile, v)
```

Comment:
We should bring this up again at code review because it keep coming up. The short term solution might be something that looks like this:
```
vcfWriter := fileio.EasyCreate(outFile)
header := vcf.NewHeader("nameFastaSample")
vcf.NewWriteHeader(vcfWriter, header)
vcf.WriteVcfToFileHandle(vcfWriter, v)
vcfWriter.Close() //defer vcfWriter.Close() at the top also works
```

I think most people like one wrap function to easily call, so long term vcf.Write() should be something like this:
`vcf.Write(filename string, header *vcf.VcfHeader, variants []*vcf.Vcf)`
```
func Write(filename string, header *VcfHeader, variants []*Vcf) {
	vcfWriter := fileio.EasyCreate(filename)
        WriteHeader(vcfWriter, header)
        WriteVcfToFileHandle(vcfWriter, variants)
        vcfWriter.Close()
}
```